Lib/test/certdata/make_ssl_certs.py

```diff
@@ -266,6 +283,11 @@ def write_cert_reference(path):
         f.write(key)
         f.write(cert)
 
+    cert = extract_cert(cert, 0)
+    with open('cert3.pem', 'w') as f:
+        f.write(key)
+        f.write(cert)
```

Comment:
It might be even easier to simply write `cert` from the previous certificate to `cert3.pem`. It will then contain the comments above the certificate, but that shouldn't prevent it to work.

Reply:
I've tried that in #124599.